### DISCUSSION TITLE
Crowdin or PR

### DISCUSSION THREAD

**Discussion by mywalkb** [mywalkb/LSPosed_mod#48] [Polls]
Today I received the first PR, for a translation, thanks @VD171, so I remembered that I still had the Crowdin link of the original LSPosed. 
I ask to the community, which is growing:

**Comment by VD171** [comment]
Creating a new pull request, needs to watch the commits of strings.xml, fork the project, check changes, edit the file and create the pull request.
Using Crowdin, you get notifications for new strings and edit them easily.

So, I prefer Crowdin.

Thank you very much for good work ❤️

**Comment by mywalkb** [comment]
I integrated crowdin in LSPosed_mod, I updated the README with link.
I hope is correctly confgured.

**Comment by VD171** [comment]
Why not create a new project in Crowdin specific for LSPosed_mod?
Since there are different strings and there will definitely be more in the future.

**Reply by mywalkb** [reply]
The new link https://crowdin.com/project/lsposedmod is a new project.
Where did you see the old project? It's impossible for me to use it, crowdin doesn't have the ability to fork like with github, so I had to rebuild it from the beginning.

**Reply by VD171** [reply]
https://github.com/mywalkb/LSPosed_mod/blob/main/README.md?plain=1#L59

**Reply by mywalkb** [reply]
Thank so much for the feedback, I fixed the link.

**Reply by VD171** [reply]
Nice! I've joined to project. May you add me as editor/proofreader for both Portuguese translations, please?

**Reply by mywalkb** [reply]
Done, could you confirm if works? So I learn to understand how to use crowdin.
In an attempt to map the pt-BR language, now all the strings are translated.

**Reply by VD171** [reply]
Yes, I got the notification, thank you so much. I can see, I will keep watching for new strings. Let's keep LSPosed alive and growing :-) Thank you very much for good work ❤️

**Comment by mywalkb** [comment]
All wrong approved strings are now reverted.
Now users can translate without issues.
Thank you @VD171 and @juvannx for your contribute.
At the moment the synchronization is manual, I need to understand why I don't receive notifications about new translations even if they apparently seem enabled.

**Comment by VD171** [comment]
Good work !
Thank you very much ❤️

**Comment by comradekingu** [comment]
@mywalkb
I think going with Weblate is the better option.
Crowdin is closed source SaaS, and it has heaps of issues.
Weblate isn't perfect, but it is the best tool for the job by a lot.

**Reply by mywalkb** [reply]
I'll study it and give it a chance
